src/50_Visual_Effects/Scroll_to_top.html

```diff
@@ -108,26 +140,17 @@
            }
          ]
      }
-   </script>
- </amp-animation>
-  <!-- We use `amp-position-observer` to start the animation when the user starts to scroll. -->
- <div id="marker">
-   <amp-position-observer
-      on="enter:hideAnim.start; exit:showAnim.start"
-      layout="nodisplay">
-    </amp-position-observer>
-  </div>
-  <!-- -->
-  <p>
-    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer sollicitudin tempor viverra. Vivamus mauris erat, euismod et viverra sed, pellentesque ut ipsum. Duis felis neque, interdum eu eros sed, egestas porttitor magna. Mauris nibh elit, pellentesque id suscipit a, fermentum id massa. Aliquam viverra ipsum lacus, eget dapibus magna scelerisque ut. Cras id augue et lacus varius laoreet. Curabitur tincidunt orci ut ex consectetur lacinia. Etiam fringilla, orci sit amet congue sollicitudin, erat purus pharetra tortor, eget efficitur nibh odio quis augue.
-
-    Etiam sed quam orci. Maecenas consectetur, libero id pharetra bibendum, turpis dolor euismod sapien, quis lacinia mi orci volutpat eros. Morbi rutrum faucibus eros eget congue. In ac efficitur ex. Morbi elementum tempor mauris lobortis tempor. Praesent lacus augue, ultrices vitae ante in, commodo accumsan odio. Cras fringilla tristique lacus, sit amet sodales leo porttitor eget. Nunc molestie sollicitudin lorem, at mollis magna elementum vel.
-
-    Aliquam tincidunt, quam nec fermentum viverra, orci dolor hendrerit ex, at suscipit tellus quam eu enim. Phasellus mollis dapibus nunc, vel sollicitudin dui consectetur id. Aliquam erat volutpat. Nunc arcu massa, fermentum in mauris sit amet, pulvinar feugiat tellus. Nam pretium tempus tortor sit amet venenatis. Aliquam hendrerit porta augue. Sed velit enim, rutrum vitae posuere at, suscipit in turpis. Sed porta diam sed arcu semper sollicitudin. Ut ac leo ut orci malesuada lobortis nec ac lorem. Cras nec turpis tellus. Nam vestibulum egestas ligula. Aliquam leo tortor, varius a iaculis vel, laoreet et neque. Vivamus sed mauris at sapien varius ultrices. Aliquam aliquam tincidunt ante, semper gravida nisl maximus vitae. In consectetur eleifend magna eu tincidunt.
-
-  </p>
-  <!-- We use the `scrollTo` action to scroll the page when the button is tapped. Find more about actions [here](https://github.com/ampproject/amphtml/blob/master/spec/amp-actions-and-events.md).-->
-  <button id="scrollToTopButton" on="tap:top-page.scrollTo(duration=200)" class="scrollToTop">⌃</button>
+    </script>
+  </amp-animation>
+  <!--     
```

Comment:
there is no header for this section

Reply:
Where it says "Here is a dummy spacer." there is no header. My understanding is that each comment needs an header, if it doesn't than this is fine.

Reply:
Every significant section/sample should have a header - the spacer is not important.

Reply:
ok I see1